llvm/test/MC/Disassembler/AMDGPU/decode-err.txt

```diff
@@ -1,17 +1,19 @@
-# RUN: llvm-mc -triple=amdgcn -mcpu=gfx900 -disassemble -show-encoding < %s 2>&1 | FileCheck -check-prefix=GCN %s
+# RUN: llvm-mc -triple=amdgcn -mcpu=gfx900 -disassemble -show-encoding < %s 2>&1 >/dev/null | FileCheck -check-prefix=GCN-ERR %s
```

Comment:
Can you use -filetype=null instead of redirect stdout to /dev/null

Reply:
Done.